hpc/app/workload_managers.yml

```diff
@@ -43,8 +43,42 @@ node_types:
         required: false
         type: string
         default: ""
+      ssh_host: 
```

Comment:
As I understood, the ssh_host is the address of the front-end host. If so, this property is unnecessary, since the address to the front-end is set in the public_address attribute. This attribute is inherited from tosca.nodes.Compute and hence not defined in this node type.